db.json

```diff
@@ -3930,7 +4023,7 @@
   "application/vnd.ms-excel": {
     "source": "iana",
     "compressible": false,
-    "extensions": ["xls","xlm","xla","xlc","xlt","xlw"]
+    "extensions": ["xls","xlm","xla","xlc","xlt","xlw","xll","xld"]
   },
```

Comment:
https://www.iana.org/assignments/media-types/application/vnd.ms-excel is very strange